AirCasting/CoreData/UIStorage.swift

```diff
@@ -0,0 +1,69 @@
+// Created by Lunar on 10/02/2022.
+//
+
+import CoreData
+import Combine
+import SwiftUI
```

Comment:
do we use `SwiftUI` here? (and `Combine`?)

Reply:
😬 copying leftover